gui/WorldView.cxx

```diff
@@ -85,10 +87,47 @@
 #include <QToolButton>
 #include <QWidgetAction>
 
+#include <array>
+
 using namespace LandmarkArrays;
 
 QTE_IMPLEMENT_D_FUNC(WorldView)
 
+
+namespace {
+vtkSmartPointer<vtkColorTransferFunction> setupBluesColorPalette(double* range)
+{
+  auto lut = vtkSmartPointer<vtkColorTransferFunction>::New();
+  std::array<std::array<double, 3>, 17> blues = {{
```

Comment:
`std::array` almost seems like overkill here, especially as it forces declaring the number of colors up front.

BTW... this is to colorize points with only one channel?

Reply:
Yes.